src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/InteractiveWindow_OutOfProc.cs

```diff
@@ -7,9 +7,10 @@ namespace Microsoft.VisualStudio.IntegrationTest.Utilities.OutOfProcess
     /// <summary>
     /// Provides a means of interacting with the interactive window in the Visual Studio host.
     /// </summary>
-    public abstract class InteractiveWindow_OutOfProc : OutOfProcComponent
+    public abstract class InteractiveWindow_OutOfProc : TextViewWindow_OutOfProc
     {
         private readonly InteractiveWindow_InProc _inProc;
+        internal override TextViewWindow_InProc InProc => _inProc;
```

Comment:
Consider the alternate pattern where you have the member here call the base type and downcast. The only scary bit here is it means the base constructor is running and then could call back into this derived one where it's not initialized yet.